### DISCUSSION TITLE
Docker compose install error

### DISCUSSION THREAD

**Discussion by bbarclay** [apache/superset#18351] [Q&A / Help]
https://issueexplorer.com/issue/apache/superset/16782

When installing via docker-compose it goes into an infant loop failing over and over. Everything mentioned in the post.

**Comment by aniaan** [comment]
Can you describe your steps in detail?

**Comment by bbarclay** [comment]
Install the master repo with docker-compose and try to enable Prophet. Prophet allows you to use predictive analytics. I've tried what seems like everything to get Prophet to install. I've added these files in the docker requirements text as suggested. And have tried installing them manually in the terminal, with different versions. I've tried uninstalling Pystan and reinstalling with an earlier version and still nothing works. 

<img width="929" alt="Screen Shot 2021-11-22 at 10 08 55 PM" src="https://user-images.githubusercontent.com/374088/142974220-b1ac7208-0a73-4564-86a9-581cbdfafe70.png">
<img width="890" alt="Screen Shot 2021-11-22 at 10 09 13 PM" src="https://user-images.githubusercontent.com/374088/142974225-fa384670-2d5a-4f4f-a0d5-ae5706f4ff33.png">

**Comment by aniaan** [comment]
The installation should install correctly, write the dependencies in `requirements-local.txt` and check the logs by executing `docker-compose logs -f superset` to see if the dependencies have not been downloaded due to network reasons

**Comment by bbarclay** [comment]
It works when I install on my Mac. But if you could shoot a small video with it working in your local, with the master branch. That would be super helpful. I've tried everything. Doing the regular from scratch install works. But something in that docker files must be wrong.
